APMrover2/system.cpp

```diff
@@ -115,7 +115,8 @@ void Rover::init_ardupilot()
     ins.set_log_raw_bit(MASK_LOG_IMU_RAW);
 
     set_control_channels();  // setup radio channels and outputs ranges
```

Comment:
Need to remove this line as it's now called in RC_Channels init method.

Reply:
Done, thanks.